Remove TXINGs check and modified to run under RH8

Gregg140 opened the pull request:
lr and all associated Load Review programs were modified to run under
RH8.

lr was  modified to eliminate the check for the TXINGS commands
as part of the Perigee Passage ECS measurements.

Commit:
Remove TXINGs check and modified to run under RH8

Commit:
Release Notes for PR 34

Commit:
Release Notes modified to add that all programs were updated to run under Python3

Commit:
Release Notes modified

cegrant commented:
I thought we were deleting the copy to luke-v from history-files altogether?
Line 197 of history-files points to a specific version of ACE-update, rather than the linked one.  Is that intentional?
Otherwise, I think this looks OK.

jazan12 reviewed (approved):
I had one small comment for you to double-check in there and noticed as I see Catherine had pointed out already the question of whether or not we are copying to luke-v or not.  Otherwise, looks good.  I'm clicking approve under assumption that either both those are as intended or the trivial fix will be done.

Gregg140 commented:
Good catch on the hard coded call to ACE-update when running in test mode. It's in the productin version also. I'll remove it.

Removing the copies to luke-v wasn't done because I was too far along in testing to add it when we decided to eliominate the copy. I can add it now since I have to re-test everything.

Commit:
Removed history files copies to luke-v and han-v

Commit:
Removed history file copying to luke-v and han-v, print model execution line,removed call to fptemp_sens_finder.pl, Updated LRHS for RH8/Python3

Gregg140 commented on history-files.pl:
```diff
@@ -822,8 +973,8 @@ ()
-	
-	$ACE_UPDATE="${test_dir}/ACE-update.pl -test $ops_dir";
+
+	$ACE_UPDATE="${test_dir}/ACE-update-v2.15.pl -test $ops_dir";
```
Fixed

Gregg140 commented:
Please re-review.

Additional changes were made: 

1) LRHS - updated to Python3
2) Removed the call to fptemp_sens_finder.pl
3) Print a line in the load review log indicating what thermal model is about to be run
4) History files were no longer copied  to luke-v and
                         han-v
lr and the associated Load Review programs, shown below, were modified, as
necessary, and tested,  to run under RH8 and Python3.

In addition, the following updates were made:

lr -  eliminated the check for the TXINGS commands as part of the
      Perigee Passage ECS measurements.
   - Removed code that called fptemp_sens_finder.pl; function no
     longer needed

LRHS - Commented statements removed.

run_models.pl - removed the call to make_dhheater_history.csh as it's
                        no longer needed.
		      - Print a line indicating what thermal model is about to be run

acis-backstop.pl - Perigee Passage TXING Quiet checks removed

history-files.pl - history files were no longer copied  to luke-v and
                         han-v

ACE-update.pl - history files were no longer copied  to luke-v and
                         han-v

Commit:
Update to the release notes

jazan12 commented on ACE-update.pl:
```diff
@@ -0,0 +1,549 @@
+
+# FLU-MON directory definitions
+
+$han_flu=GetFluMon('han-v');
```
Do you still want this defined?

Gregg140 replied:
Commented out. Might want han-v to be defined in the future.

jazan12 commented on ACE-update.pl:
```diff
@@ -0,0 +1,549 @@
+    &chgrp_by_name("acisops","$dir/$key.dat");
+}
+
+$HEAD_flu = "han-v:$xcanuck_flu"; # Global history file directory
```
Looks like you're using han-v for the global; is that still how you want to do it, or should han-v be excised altogether?

jazan12 commented on history-files.pl:
```diff
@@ -102,13 +127,16 @@
 
 $colossus_v_flu=GetFluMon('colossus-v');
-$acisocc_v_flu=GetFluMon('acisocc-v');
+#$han_v_flu=GetFluMon('han-v');
```
Do you want these definitions here for luke-v and han-v?  I don't think they're needed now, right?

Gregg140 replied:
Note that it is commented out. Keeping the line in there to get flumon file targets in case it's needed in the future.

jazan12 reviewed (approved):
I see some definitions that look extraneous that I think could be deleted but nothing that looks like it would be problematic to run.

Commit:
Commented out an unused call to GetFluMon for han-v

Commit:
Updated Release_Notes_V4.3.txt

Commit:
Updated Release_Notes_V4.3.txt
